package.json

```diff
@@ -27,25 +27,35 @@
   ],
   "dependency-check": {
     "ignore": [
-      "react-dom",
+      "@hot-loader/react-dom",
```

Comment:
what does this package do again?

Reply:
`@hot-loader/react-dom` is normal `react-dom` with some patches applied to be more friendly for using:
https://github.com/hot-loader/react-dom
Webpack 4 throws `"React-Hot-Loader: react-🔥-dom patch is not detected. React 16.6+ features may not work."` error in console. To get read of it and provide a support latest features of React regular `react-dom` package was replaced with `@hot-loader/react-dom`.

Reply:
awesome! should we update our compass template for this as well! probably will also throw on all the newly created plugins

Reply:
already done :)